packages/ihe-gateway-sdk/src/models/shared.ts

```diff
@@ -65,27 +66,25 @@ export const operationOutcomeSchema = z.object({
 });
 export type OperationOutcome = z.infer<typeof operationOutcomeSchema>;
 
-export const xcpdPatientIdSchema = z.object({
+export const externalGatewayPatientSchema = z.object({
   id: z.string(),
   system: z.string(),
 });
-export type XCPDPatientId = z.infer<typeof xcpdPatientIdSchema>;
+export type XCPDPatientId = z.infer<typeof externalGatewayPatientSchema>;
 
 export const baseResponseSchema = z.object({
   id: z.string(),
   timestamp: z.string(),
   responseTimestamp: z.string(),
-  externalGatewayPatient: xcpdPatientIdSchema.optional(),
+  cxId: z.string().optional(),
+  externalGatewayPatient: externalGatewayPatientSchema.optional(),
   patientId: z.string().optional(),
 });
 export type BaseResponse = z.infer<typeof baseResponseSchema>;
 
-export const baseErrorResponseSchema = z.intersection(
-  baseResponseSchema,
-  z.object({
-    operationOutcome: operationOutcomeSchema,
-  })
-);
+export const baseErrorResponseSchema = baseResponseSchema.extend({
+  operationOutcome: operationOutcomeSchema,
```

Comment:
I  think this might be optional

Reply:
AFAIK if there is an error there needs to be an operationOutcome i didnt see any where in the notion docs stating otherwise for faulty responses

Reply:
Hmmm yah I don't really know where shamil got all the info about what was required or not. We can stick with required